Makefile

```
@@ -46,12 +46,19 @@ KO_PREFIX ?= gcr.io/k8s-staging-zeitgeist
 KO_DOCKER_REPO=$(KO_PREFIX)
 
 build: ## Build zeitgeist
+	# build local version
 	go build -trimpath -ldflags "$(LDFLAGS)" -o ./output/zeitgeist .
+	# build remote version
+	go build -trimpath -ldflags "$(LDFLAGS)" -o ./output/zeitgeist ./remote/zeitgeist
```

```suggestion
	go build -trimpath -ldflags "$(LDFLAGS)" -o ./output/zeitgeist-remote ./remote/zeitgeist
```

otherwise it will overwrite the build that is done above